helpers/test/getLocalizeResources.test.js

```diff
@@ -208,7 +208,8 @@ describe('getLocalizeResources', () => {
 
 		expect(formatFuncSpy).to.have.been.callCount(1); // 1 lms
 		expect(formatFuncSpy).to.have.been.calledWithExactly();
-		expect(openSpy).to.have.been.calledOnceWithExactly('d2l-oslo');
+		expect(openSpy).to.always.have.been.calledWithExactly('d2l-oslo');
+		expect(openSpy).to.have.callCount(3);
```

Comment:
So seems like this increases how many times the cache gets opened. This seems not ideal but not sure how else we can verify if CacheStorage is available to the browser.

Reply:
Yeah in my experience I haven't seen a way to check if local storage is available without trying to write/read to it.